app/views/search/profiles.html.erb

```diff
@@ -2,6 +2,12 @@
 
 <div class="container">
   <% if @profiles.present? %>
+<div class="form-group form-inline">
+    <%= form_tag request.url, :method => 'get' do %>
+        <%= text_field_tag :tag, params[:tag], id:'tag', placeholder: "Filter by user tag"%>
```

Comment:
ah, i think we can actually use the `query` parameter as the term by which to search user tags as well -- make sense?  I'll clarify in the tests below.